lib/parser.js

```diff
@@ -0,0 +1,21 @@
+var falafel = require('falafel');
+var esprima = require('esprima');
+
+module.exports = {
+  walk: function(js, callback) {
+    return falafel(js, {
+      parser: this
+    }, function(node) {
+      node.start = node.range[0];
```

Comment:
This is probably alright to leave here, but if you could add a comment stating that fact.